data/use_cases.yml

```
@@ -3,19 +3,39 @@
 developer_onboarding:
   title: Developer Onboarding
   link: /strategy-goals/strategy/use_cases/dev-onboarding
+  sponsors:
+    product: ben_venker
+    engineering: robert_lin
```

@bobheadxi @tsenart - both of you signed up for two areas, but we want just one sponsor each so there's more people participating. Can you remove yourself from one or the other, and keep whichever one is your favorite?

---

Done https://github.com/sourcegraph/handbook/pull/2447/files#r817628443

---

@bobheadxi I removed you from developer onboarding since I haven't heard from you - if you want to switch to code reuse please feel free to do so!

```suggestion
    engineering: 
```

---

Soundsd good!